cedar-drt/fuzz/src/schemas.rs

```diff
@@ -105,7 +105,7 @@ fn action_type_equivalence(name: &str, lhs: ActionType, rhs: ActionType) -> Resu
             (Some(lhs), Some(rhs)) => {
                 // If either of them has at least one empty appliesTo list, the other must have the same attribute.
                 // Otherwise both of them must apply to unspecified entities or non-empty entity lists, which must be equal.
-                if (either_empty(&lhs) && either_empty(&rhs)) || rhs == lhs {
+                if rhs == lhs {
```

Comment:
I imagine this comment needs to be updated if this guard is removed. Likely some other comments in this file need updating too.

Also: `either_empty` is checking for empty lists, which are used for action groups -- why do these checks need to be removed for RFC55?